### PR TITLE
New minor release of the web programming language Links, version 0.7.3.

### PR DESCRIPTION
**Pull request by dhil**
The glorious bits

- Improved dynamic loading of available database drivers
- Compatibility with OCaml 4.06+ compiler tool-chain
- Various bug fixes

### CODE REVIEW

File: packages/links/links.0.7.3/opam [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+available: [
+  ocaml-version >= "4.06.0"
+]
+
+build: [
+  [ "jbuilder" "subst" "-n" name ] {pinned}
+  [ "jbuilder" "build" "-p" name "-j" jobs ]
+]
+
+install: [
```
Comment (kit-ty-kate) [inline packages/links/links.0.7.3/opam]: Have you not found any ways of installing those files using `jbuilder`'s `install` stanza ?

Comment (dhil) [reply packages/links/links.0.7.3/opam]: We are not aware of any way to copy those files and directories programmatically using jbuilder's stanzas.

Comment (samoht) [reply packages/links/links.0.7.3/opam]: /cc @diml and @rgrinberg

Comment (diml) [reply packages/links/links.0.7.3/opam]: Currently we can't put globs in `install` stanzas. As long as these don't include generated files you could either use the promote feature to maintain a generated and committed jbuild file, or use [the OCaml syntax](https://dune.readthedocs.io/en/latest/jbuild.html#id4).

If not done already, please open a ticket on ocaml/dune about this with your use case.

**Comment by kit-ty-kate** [comment]
Sorry for the delay. Thanks!

**Pull request merged**